include/net/net_stats.h

```diff
@@ -323,6 +327,10 @@ struct net_stats {
 	/** Network packet TX time detail statistics */
 	struct net_stats_tx_time tx_time_detail[NET_PKT_DETAIL_STATS_COUNT];
 #endif
+#if defined(CONFIG_NET_PKT_RXTIME_STATS_DETAIL)
+	/** Network packet RX time detail statistics */
+	struct net_stats_tx_time rx_time_detail[NET_PKT_DETAIL_STATS_COUNT];
```

Comment:
This should likely be `struct net_stats_rx_time`.

Also it looks like this field will be only visible if `CONFIG_NET_PKT_TXTIME_STATS` is selected as well due to
```
#if defined(CONFIG_NET_CONTEXT_TIMESTAMP) || \
					defined(CONFIG_NET_PKT_TXTIME_STATS)
```
a few lines above.

Reply:
Good points, I will send a fix for these.